doc/user-manual.md

```diff
@@ -1,18 +1,264 @@
+$ java "-javaagent:${EASE_AGENT_PATH}/easeagent.jar -Deaseagent.log.conf=${EASE_AGENT_PATH}/log4j2.xml" -jar user-app.jar
+```
+
+## Prometheus Support
+When Internal HTTP Server is enabled, User can use Prometheus to collect metrics information.
+* Adding the following configuration in `prometheus.yml`
+```
+  - job_name: 'user-app'
+    static_configs:
+    - targets: ['localhost:9900']
+    metrics_path: "/prometheus/metrics"
+```
+
+## Health Check and Readiness Check Endpoint
+EaseAgent supply the `health check`、`readiness check` endpoint.
+
+* `Health Check` Endpoint
+```
+[GET] http://[ip]:[easeagent.server.port]/health
+The response status will be 200(OK)
+```
+
+* `Readiness Check` Endpoint
+After Spring sending `ApplicationReadyEvent`, EaseAgent will change readiness status to `true`
+```
+[GET] http://[ip]:[easeagent.server.port]/health/readiness
+The response status will be 200(OK)
+```
+
+## Tracing
+EaseAgent use [brave](https://github.com/openzipkin/brave) to collect tracing logs.The data format stored in `Kafka`  is [Zipkin Data Model](https://zipkin.io/pages/data_model.html). User can send tracing logs to [Zipkin server](https://zipkin.io/pages/quickstart.html).
+
+### Tracing Component
+Component Type | Component | Reference | 
+--- | ---| --- |
+HTTP Client | `RestTemplate`、 `WebClient`、 `FeignClient` | [brave-instrumentation-http](https://github.com/openzipkin/brave/tree/master/instrumentation/http)
+HTTP Server | `Servlet`、`Filter` | [brave-instrumentation-http](https://github.com/openzipkin/brave/tree/master/instrumentation/http)
+DataBase | `JDBC` | [Brave](https://github.com/openzipkin/brave/tree/master/brave)
+Cache | `Jedis`、`Lettuce` | [Brave](https://github.com/openzipkin/brave/tree/master/brave)
+Message | `RabbitMQ`、`Kafka` | [brave-instrumentation-messaging](https://github.com/openzipkin/brave/tree/master/instrumentation/messaging) 、[Brave Kafka instrumentation](https://github.com/openzipkin/brave/tree/master/instrumentation/kafka-clients)
+Logging | `Log4j2`、`Logback` | [brave-context-log4j2](https://github.com/openzipkin/brave/tree/master/context/log4j2) 、[brave-context-slf4j](https://github.com/openzipkin/brave/tree/master/context/slf4j)
 
-The metric name's rule are as follows:
+### Custom Span Tag
 
+#### JDBC
+Tag | Description |
+--- | ---|
+sql | Sql text in user application |
+local-component | Default value = 'database' |
+url | Connection information. Example: `jdbc:mysql://localhost:3306/db_demo` |
+error | SQLException information |
+
+#### Cache
+Tag | Description |
+--- | ---|
+redis.method | Redis command. Example: `MGET`、`GET` |
+
+#### RabbitMQ Producer And Consumer
+Tag | Description |
+--- | ---|
+rabbit.exchange | RabbitMQ exchange |
+rabbit.routing_key | RabbitMQ routingKey |
+rabbit.queue | RabbitMQ routingKey |
+
+#### Kafka Producer And Consumer
+Tag | Description |
+--- | ---|
+kafka.key | Kafka consumer record Key |
+kafka.topic | Kafka topic |
+kafka.broker | Kafka url |
+
+## Metric
```

Comment:
I need to tell users that we pack the metrics value in what's packge format/protocol

Reply:
> We need to tell users that we pack the metrics value in what's packge format/protocol

I will move Metric Info to `Developement-guide` and fix it